docs/UserGuide.md

```diff
@@ -44,7 +44,7 @@ commands that you can execute.
 
 You should see the following output when you start NUSBuddy for the first time.
 
-```shell
+```
```

Comment:
Thanks for fixing this. Remember to close the corresponding issues related to this (warning there are a lot!).